2016/proposals/result-aggregation-server-garg-prerit.md

```diff
@@ -89,16 +90,29 @@ An API is to be developed to store the results generated by the test script. I s
 This would involve : 
 * **Modification of the installation-test-scrips** and giving the user an option to send the data to the server by making a [POST](https://en.wikipedia.org/wiki/POST_(HTTP)) request to the API endpoint with payload of all the information collected. Code Snippet : 
 ```python
-import requests 
- 
-headers = { 
-    "Content-Type": "application/json", 
-} 
-HOST = "http://127.0.0.1:5000" 
-os_end_point = "/system_info/" 
-data = {"system_dist" : system_dist , "version": version} 
-uri = '{}{}'.format(HOST, os_end_point)
-response = requests.post(uri, data=json.dumps(data), headers=headers) 
+python_version = platform.python_version()[0]
+if python_version == '2':
+  import httplib
+elif python_version == '3':
+  import http.client
```

Comment:
In Python such conditionals are done by duck typing (as in here: http://stackoverflow.com/a/17029047).